modules/trino/src/main/java/org/testcontainers/containers/TrinoContainer.java

```diff
@@ -0,0 +1,92 @@
+package org.testcontainers.containers;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.VisibleForTesting;
+import org.testcontainers.utility.DockerImageName;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.google.common.base.Strings.nullToEmpty;
+import static java.lang.String.format;
+
+/**
+ * @author findepi
+ */
```

Comment:
nit: we're not using the `@author` tag and rely on GitHub's contributor stats :)
```suggestion
```

Reply:
i thought this was the project convention
https://github.com/testcontainers/testcontainers-java/blob/672cbeebcac6310caab90383bce11a0b753b1106/core/src/main/java/org/testcontainers/containers/wait/strategy/HostPortWaitStrategy.java#L19
et al

Reply:
There are _some_ `@author` tags in a couple of places (especially some very old code) but we agreed to not use them (and we need to make it consistent), and they are mostly there because IDEA generated them :)